testing/test_terminal.py

```diff
@@ -2408,6 +2408,43 @@ def test_foo():
             )
         )
 
+    def test_code_highlight_custom_theme(
+        self, pytester: Pytester, color_mapping, monkeypatch: MonkeyPatch
+    ) -> None:
+        pytester.makepyfile(
+            """
+            def test_foo():
+                assert 1 == 10
+        """
+        )
+        monkeypatch.setenv("PYTEST_THEME", "solarized-dark")
+        result = pytester.runpytest("--color=yes")
+        result.stdout.fnmatch_lines(
+            color_mapping.format_for_fnmatch(
```

Comment:
i think we can either drop call  to the color mapping formatter or add a mode where we can enter actual expected numbers for a color scheme (to read better than the raw escape sequences)